Dockerfile

```diff
@@ -1,3 +1,5 @@
+# syntax=docker/dockerfile:experimental
```

Comment:
Might want to specify the version here so you don't wind up with some accidental breakage in the future.